integration-tests/pom.xml

```diff
@@ -14,6 +14,11 @@
   <version>0.0.1-SNAPSHOT</version>
   <name>Event Bridge :: Integration Tests</name>
 
+  <properties>
+    <cucumber.version>7.0.0</cucumber.version>
```

Comment:
Please move the version definition into https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/build-parent/pom.xml.

Reply:
Okay